server/src/main/java/org/candlepin/model/IdentityCertificate.java

```diff
@@ -30,6 +30,8 @@
 
 
 
+
+
```

Comment:
If you are making a change anyway, kill off this white space -- a little too much IMO.

Reply:
Rebase artifact, I'm guessing. I typically set that to three lines between imports and the class definition.